power-management/Dockerfile.template

```diff
@@ -0,0 +1,3 @@
+FROM scratch
+# copy script to create a minimal image
+COPY start.sh .
```

Comment:
On device-types other than `amd64` this service does nothing

![image](https://user-images.githubusercontent.com/13220877/78703312-fed98400-791a-11ea-8295-11235fbea817.png)